helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -39,13 +39,23 @@ spec:
         sidecar.istio.io/inject: "false"
         {{- end }}
     spec:
+      {{- if not .Values.isWindows }}
       hostNetwork: true
+      {{- end }}
```

Comment:
This will affect host network metrics. What prevents us from keeping it? Can signalfx-agent run in hostNetwork on Windows nodes?

Reply:
We can not set `hostNetwork: true` for the windows worker node because `/etc/cni/net.d` and its `/opt/cni/bin` will note be present on the host. so we are getting the following error for that.

<img width="1792" alt="Screenshot 2021-09-20 at 10 24 42 AM" src="https://user-images.githubusercontent.com/88535006/133958742-30620641-65ea-4917-9578-62bc2013b74c.png">

Also, for SignalFx-agent we are not keeping it in case of windows installation - [Reference](https://github.com/signalfx/signalfx-agent/blob/main/deployments/k8s/helm/signalfx-agent/templates/daemonset.yaml#L38)

Reply:
ok. sounds good